ISSUE_TEMPLATE.md

```diff
@@ -14,6 +14,9 @@
 ( ) 8.0
 ( ) 8.1
 
+### Terminus Version
+Please note which version of Terminus you are using.
```

Comment:
I think the instruction can be omitted from what is seen after the issue is created.
```suggestion
<!-- Please note which version of Terminus you are using.-->
```